src/Controls/src/Core/ShellToolbar.cs

```diff
@@ -166,7 +166,7 @@ void OnBackButtonPropertyChanged(object sender, System.ComponentModel.PropertyCh
 			void OnBackButtonCanExecuteChanged(object sender, EventArgs e)
 			{
 				BackButtonEnabled =
-					_backButtonCommand.CanExecute(_backButtonBehavior.CommandParameter);
+					_backButtonCommand != null && _backButtonCommand.CanExecute(_backButtonBehavior?.CommandParameter);
```

Comment:
Can you add tests here? https://github.com/dotnet/maui/blob/cc8a118b51cd2fe9af0dc9d87933d49b4576080e/src/Controls/tests/Core.UnitTests/ShellTests.cs#L1328

Reply:
Added.
<img width="517" alt="Captura de pantalla 2022-11-22 a las 12 21 49" src="https://user-images.githubusercontent.com/6755973/203301937-57ba21d6-f6a1-4a89-a182-515250be7763.png">

Reply:
I don't know if the test is quite covering the scenario here.  If I breakpoint the offending code the test never triggers that code

![image](https://user-images.githubusercontent.com/5375137/203356513-9070c6bf-a1c4-475b-9deb-091d05dea5d7.png)